lib/structures/GuildIntegration.js

```diff
@@ -41,6 +45,8 @@ class GuildIntegration extends Base {
         this.enableEmoticons = data.enable_emoticons;
         this.subscriberCount = data.subscriber_count;
         this.syncedAt = data.synced_at;
+        this.revoked = data.revoked;
+        this.application = data.application;
```

Comment:
Discord may not send this
```suggestion
        if(data.application !== undefined) {
            this.application = data.application;
        }
```

Reply:
This is not really necessary since it would be undefined anyway, but for the sake of it we can do that.

Reply:
Just preventing an unnecessary assignment ;)